components/datadog/apps/dogstatsd/k8s.go

```diff
@@ -65,7 +67,7 @@ func K8sAppDefinition(e config.CommonEnvironment, kubeProvider *kubernetes.Provi
 							Env: &corev1.EnvVarArray{
 								&corev1.EnvVarArgs{
 									Name:  pulumi.String("STATSD_URL"),
-									Value: pulumi.StringPtr("unix:///var/run/datadog/dsd.socket"),
+									Value: pulumi.String("unix:///var/dsd.socket"),
```

Comment:
Pulumi is magic 🪄
`pulumi.String(…)` and `pulumi.StringPtr(…)` are interchangeable !